x/ibc/core/exported/client.go

```diff
@@ -32,6 +32,14 @@ type ClientState interface {
 	Validate() error
 	GetProofSpecs() []*ics23.ProofSpec
 
+	// Initialization function
+	// Clients must validate the initial consensus state, and may store any client-specific metadata
+	// necessary for correct light client operation
+	Initialize(sdk.Context, codec.BinaryMarshaler, sdk.KVStore, ConsensusState) error
```

Comment:
Needed this function to set initial metadata for tendermint clients (initial processed time) + closing #7982